frameworks/digital-outcomes-and-specialists-4/questions/declaration/mitigatingFactors3.yml

```diff
@@ -0,0 +1,10 @@
+name: Mitigating factors for Modern Slavery reporting requirements
+question: >
+  If your organisation does not meet the requirements, please explain why.
+type: textbox_large
+
+hidden: true
+validations:
+  - 
+    name: answer_required
```

Comment:
Is the line break here (after the dash) necessary? I find it a bit confusing

Reply:
Not really - it's consistent with the other `mitigatingFactors` questions though. I'm trying not to change the existing questions too much.

Reply:
Okay might as well stay consistent